.eslintrc.restrict-imports.cjs

```diff
@@ -39,6 +39,20 @@ module.exports = {
             from: ['./src/!(shared|k8s)/**/*'],
             message: 'Files in `./src/shared` may only import from `./src/shared` or `./src/k8s`.',
           },
+          // Zone 6: Files in `./src/feature-flags` may only import from `./src/feature-flags` or `./src/components`.
+          {
+            target: './src/feature-flags/**/*',
+            from: ['./src/!(feature-flags|components)/**/*'],
```

Comment:
I dont think we need to import anything from the components directory, createModallauncher should be in `shared` folder, for now we can add rule.
```
'./src/!(feature-flags|components/modal)/**/*'